README.md

```diff
@@ -118,6 +145,62 @@ TODO
 TODO
 -->
 
+## GridMaterial
+
+_extends THREE.MeshPhongMaterial_
+
+A material with the same interface as `THREE.MeshPhongMaterial`. It's used to add a grid on the mesh, created by the method `csgGridShaderMixin` passing a shader. Default is enabled but if you want you can disable the grid with
+```js
+gridMat.enableGrid = false;
+```
```

Comment:
Here we can add a member section for `enableGrid` that describes its use like so:

````md
### .enableGrid

```js
enableGrid = false : Boolean
```

TODO description
````